project.toml

```diff
@@ -0,0 +1,16 @@
+[_]
+schema-version = "0.2"
+
+[io.buildpacks]
+# Exclude files from local Pack CLI builds, where .gitignore doesn't apply.
+
+exclude = [
```

Comment:
(not blocking) I believe Kodon doesn't yet support the `exclude` directive - I don't suppose after this PR you could reach out to the Builds team and make sure they have something in their backlog for adding support for it? :-)